addons/web/static/src/views/widgets/ribbon/ribbon.scss

```diff
@@ -0,0 +1,74 @@
+.ribbon {
```

Comment:
css duplicated, is it a good idea?
solutions:
 - change className
 - move file instead of duplicating it
 
 I don't know